pallets/capacity/src/lib.rs

```diff
@@ -211,6 +226,36 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		// TODO: real weights
```

Comment:
Can this TODO line be removed?

Reply:
Yes I was planning to remove it once weights are finalized.